.github/workflows/PUBLISH_NPM_PACKAGE.yml

```diff
@@ -0,0 +1,61 @@
+name: Deploying npm package
+run-name: Deploying npm package version ${{ inputs.npm-package-version }}
+
+on:
+  workflow_dispatch:
```

Comment:
Should we also invoke this as part of our regular release? (e.g. as a job in the release workflow)

This can also be done on later stages ofc to keep the scope of the PR reasonable